--- sarg-2.1.orig/log.c	2006-01-04 22:47:25.000000000 +0100
+++ sarg-2.1/log.c	2006-01-09 13:37:52.000000000 +0100
@@ -124,7 +124,9 @@
    unsigned long nreg=0;
    off_t recs1=0;
    unsigned long recs2=0;
+#if HAVE_GETRLIMIT
    struct rlimit rl;
+#endif
    int OutputNonZero = REPORT_EVERY_X_LINES ;
 
    BgImage[0]='\0';
@@ -697,6 +699,7 @@
       sprintf(msg,"sarg %s: %s",text[73],VERSION);
       debuga(msg);
    }
+#if HAVE_GETRLIMIT
 #if defined(RLIMIT_NOFILE)
    getrlimit (RLIMIT_NOFILE, &rl);
 #else #if defined(RLIMIT_OFILE)
@@ -720,7 +723,7 @@
       sprintf(msg,"Maximum file descriptor: cur=%ld max=%ld, changed to cur=%ld max=%ld",l1,l2,rl.rlim_cur,rl.rlim_max);
       debuga(msg);
    }
-
+#endif
    if(UserTabFile[0] != '\0') {
       if(debug) {
          sprintf(msg,"%s: %s",text[86],UserTabFile);
