--- sarg-2.0.4-orig/decomp.c	2005-02-21 19:13:21.000000000 +0100
+++ sarg-2.0.4/decomp.c	2005-02-24 20:37:41.000000000 +0100
@@ -34,8 +34,10 @@
    }
 
    if(strstr(arq,".gz") !=0) {
-      sprintf(cmd,"%s: %s > %s/sarg-file.in (zcat)",text[62],arq,tmp);
-      debuga(cmd);
+      if (debug) {
+          sprintf(cmd,"%s: %s > %s/sarg-file.in (zcat)",text[62],arq,tmp);
+          debuga(cmd);
+      }
       sprintf(cmd,"zcat %s > %s/sarg-file.in",arq,tmp);
       system(cmd);
       strcpy(zip,"zcat");
@@ -44,8 +46,10 @@
    }
 
    if(strstr(arq,".bz2") != 0) {
-      sprintf(cmd,"%s: %s > %s/sarg-file.in (bzcat)",text[62],arq,tmp);
-      debuga(cmd);
+      if (debug) {
+	  sprintf(cmd,"%s: %s > %s/sarg-file.in (bzcat)",text[62],arq,tmp);
+	  debuga(cmd);
+      }
       sprintf(cmd,"bzcat %s > %s/sarg-file.in",arq,tmp);
       system(cmd);
       strcpy(zip,"zcat");
@@ -54,8 +58,10 @@
    }
 
    if(strstr(arq,".Z")) {
-      sprintf(cmd,"%s: %s (uncompress)",text[62],arq);
-      debuga(cmd);
+      if (debug) {
+	  sprintf(cmd,"%s: %s (uncompress)",text[62],arq);
+	  debuga(cmd);
+      }
       sprintf(cmd,"uncompress %s",arq);
       system(cmd);
       arq[strlen(arq)-2]='\0';
--- sarg-2.0.4-orig/log.c	2005-02-22 15:06:18.000000000 +0100
+++ sarg-2.0.4/log.c	2005-02-24 20:38:42.000000000 +0100
@@ -765,8 +765,10 @@
          recs1++;
       rewind(fp_in);
 
-      printf("SARG: Records in file: %d, reading: %3.2f%%\r",recs1,(float) 0);
-      fflush( stdout ) ;
+      if (debug) {
+	printf("SARG: Records in file: %d, reading: %3.2f%%\r",recs1,perc);
+	fflush (stdout);
+      }
    }
    
     
