diff -dur sarg-2.0.1.orig/sarg.1 sarg-2.0.1/sarg.1
--- sarg-2.0.1.orig/sarg.1	2004-10-03 23:37:03.000000000 +0200
+++ sarg-2.0.1/sarg.1	2004-11-02 15:09:38.889512573 +0100
@@ -131,11 +131,11 @@
 .B \-split
 Split the log file by date in \fB-d\fP parameter.
 .SH FILES
-.BR /usr/local/sarg/sarg.conf
+.BR /etc/sarg/sarg.conf
 .br
-.B /usr/local/squid/logs/access.log
+.B /var/log/squid/access.log
 .br
-.B /usr/local/squidGuard/logs/squidGuard.log
+.B /var/log/squidGuard/squidGuard.log
 .SH SEE ALSO
 squid(8)
 .SH AUTHOR
diff -dur sarg-2.0.1.orig/sarg.conf sarg-2.0.1/sarg.conf
--- sarg-2.0.1.orig/sarg.conf	2004-10-03 23:37:03.000000000 +0200
+++ sarg-2.0.1/sarg.conf	2004-11-02 15:11:25.379179337 +0100
@@ -29,7 +29,7 @@
 #       Where is the access.log file
 #       sarg -l file
 #
-#access_log /usr/local/squid/var/logs/access.log
+access_log /var/log/squid/access.log
 
 # TAG: graphs yes|no
 #	Use graphics where is possible.
@@ -136,7 +136,7 @@
 #       The reports will be saved in that directory
 #       sarg -o dir
 #
-#output_dir /var/www/html/squid-reports
+output_dir /var/lib/sarg
 
 # TAG:  output_email
 #       Email address to send the reports. If you use this tag, no html reports will be generated.
@@ -260,7 +260,7 @@
 # TAG: exclude_codes file
 #      Ignore records with these codes. Eg.: NONE/400
 #
-#exclude_codes /usr/local/sarg/exclude_codes
+exclude_codes /etc/sarg/exclude_codes
 
 # TAG: replace_index string
 #      Replace "index.html" in the main index file with this string
@@ -427,7 +427,7 @@
 #      path to squidGuard.conf file
 #      Generate reports from SquidGuard logs.
 #      Use 'none' to disable.
-#      squidguard_conf /usr/local/squidGuard/squidGuard.conf
+#      squidguard_conf /var/log/squidGuard/squidGuard.conf
 #
 #squidguard_conf none
 
@@ -481,7 +481,7 @@
 #     Sarg will create sarg-php directory with some PHP modules:
 #     - sarg-squidguard-block.php - add urls from user reports to squidGuard DB
 #
-#www_document_root /var/www/html
+www_document_root /var/lib/sarg
 
 # TAG block_it module_url
 #     This tag allow you to pass urls from user reports to a cgi or php module,
@@ -524,7 +524,7 @@
 #                                          %u - user report
 #
 # user_authentication no
-# AuthUserFile /usr/local/sarg/passwd
+# AuthUserFile /etc/sarg/passwd
 # AuthName "SARG, Restricted Access"
 # AuthType Basic
 # Require user admin %u
